scripts/verify.sh

```diff
@@ -41,7 +41,9 @@ golangci-lint run --disable-all \
     --enable=dupl \
     --enable=goimports \
     --enable=gocyclo \
-
+    --enable=staticcheck \
+    --enable=unused \
+    --enable=gosimple
```

Comment:
```suggestion
    --enable=gosimple \
```

If it accepts this ending backslash, I think this will mean less rebase needed to include the rest of the lints as other PR would not require to modify that line

Reply:
will be required to do the rebase/merge any way 👍